test/universal.js

```diff
@@ -21,87 +21,91 @@ describe('Google Analytics :: Universal', function(){
     test.mapper(mapper);
   });
 
-  describe('mapper', function(){
-    describe('page', function(){
-      it('should map basic page', function(){
+  describe('mapper', function() {
+    describe('page', function() {
+      it('should map basic page', function() {
         test.maps('page-basic', settings);
       });
 
-      it('should map context.app', function(){
+      it('should map context.app', function() {
         test.maps('page-app', settings);
       });
 
-      it('should map context.campaign', function(){
+      it('should map context.campaign', function() {
         test.maps('page-campaign', settings);
       });
 
-      it('should map context.screen', function(){
+      it('should map context.screen', function() {
         test.maps('page-screen', settings);
       });
 
-      it('should map context.locale', function(){
+      it('should map context.locale', function() {
         test.maps('page-locale', settings);
       });
 
-      it('should map page with custom dimensions and metrics', function(){
+      it('should map page with custom dimensions and metrics', function() {
         test.maps('page-cm-cd', settings);
       });
     });
 
-    describe('track', function(){
-      it('should map basic track', function(){
+    describe('track', function() {
+      it('should map basic track', function() {
         test.maps('track-basic', settings);
       });
 
-      it('should map context.app', function(){
+      it('should map context.app', function() {
         test.maps('track-app', settings);
       });
 
-      it('should map context.screen', function(){
+      it('should map context.screen', function() {
         test.maps('page-screen', settings);
       });
 
-      it('should map page with custom dimensions and metrics', function(){
+      it('should map page with custom dimensions and metrics', function() {
         test.maps('track-cm-cd', settings);
       });
+
+      it('should map url in track call', function() {
+        test.maps('track-url', settings);
+      });
     });
 
-    describe('completed-order', function(){
-      it('should map basic completed-order', function(){
+    describe('completed-order', function() {
+      it('should map basic completed-order', function() {
         test.maps('completed-order-basic', settings);
       });
 
-      it('should map context.app', function(){
+      it('should map context.app', function() {
         test.maps('completed-order-app', settings);
       });
 
-      it('should map context.screen', function(){
+      it('should map context.screen', function() {
```

Comment:
fits the style enforced in other projects, we just haven't added linting to SS ints yet